container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/logs_collector_functions/implementations/fluentbit/fluentbit_logs_collector_daemonset.go

```diff
@@ -433,7 +437,12 @@ func createLogsCollectorConfigMap(
 		labels,
 		annotations,
 		map[string]string{
-			fluentBitConfigFileName: fluentBitConfigStr,
+			fluentBitConfigFileName: fmt.Sprintf(
+				fluentBitConfigFmtStr,
+				logsCollectorHttpPortNum,
+				label_value_consts.UserServiceKurtosisResourceTypeKubernetesLabelValue.GetString(),
+				logsAggregatorHost,
+				logsAggregatorPortNum),
 		},
```

Comment:
now that aggregator is setup, we can dynamically configure collectors to send logs to it